uber/templates/groups/history.html

```diff
@@ -2,7 +2,11 @@
 {% block title %}Group History - {{ group.name }}{% endblock %}
 {% block content %}
 
-{% nav_menu group "form?id={id}" "Group Page" True "history?id={id}" "History" True "index" "Return to Group List" True %}
+{{ group.nav_menu(
```

Comment:
This is different from the previous menu (another Group menu) and the Job menu down below a bit. Those menus have this kind of syntax:
```python
{{ macros.nav_menu(
        group, c.PAGE_PATH,
        "form?id={id}", "Group Page", True,
        "history?id={id}", "History", True,
        "index", "Return to Group List", True
    ) }}
```

Note particularly the `        group, c.PAGE_PATH,` which is not here and the fact that this says `group.nav_menu` and not `macros.nav_menu`. Possibly you missed converting a menu to the new `macros` syntax?